interfaces/policy/helpers.go

```diff
@@ -66,6 +67,19 @@ func checkID(kind, id string, ids []string, special map[string]string) error {
 	return fmt.Errorf("%s does not match", kind)
 }
 
+func checkOnClassic(c *asserts.OnClassicConstraint) error {
+	if c == nil {
+		return nil
+	}
+	if c.Classic != release.OnClassic {
+		return fmt.Errorf("on-classic mismatch")
+	}
+	if c.Classic && len(c.Distros) != 0 {
+		return checkID("distro", release.ReleaseInfo.ID, c.Distros, nil)
+	}
```

Comment:
that's what checkID does I think, checking that ID is in c.Distros, I'm probably not understanding the comment